### DISCUSSION TITLE
Marketing Specialist & Ideas

### DISCUSSION THREAD

**Discussion by aalisinaa** [WeAreMahsaAmini/Core#36] [Ideas]
NA

**Comment by AnonymousMaximus2** [comment]
افرادی که در خارج به پلیس حمله میکنند از خودشان هستند و بهشون پول میدن به نام مردم می اندازند به جان پلیس تا بگویند اعتراض کننده ها بد هستند.



من یک ایده ای داده ام در لینک زیر بد نیست نگاهی بیاندازید

https://github.com/WeAreMahsaAmini/Core/discussions/115
